gateway/modules/schema/inspection.go

```diff
@@ -78,15 +76,6 @@ func generateInspection(dbType, col string, fields []model.InspectorFieldType, f
 				}
 			}
 
-			if model.DBType(dbType) == model.Postgres {
-				// split "'default-value'::text" to "default-value"
-				s := strings.Split(field.FieldDefault, "::")
-				field.FieldDefault = s[0]
-				if fieldDetails.Kind == model.TypeString || fieldDetails.Kind == model.TypeDateTime || fieldDetails.Kind == model.TypeID {
-					field.FieldDefault = strings.Split(field.FieldDefault, "'")[1]
-				}
-			}
-
```

Comment:
Are you sure we can remove this?

Reply:
Yes, To confirm you can check the Postgres query in describe table, The filtering logic is implemented in the query itself